dist/templates/rbac-ovnkube-node.yaml.j2

```diff
@@ -153,6 +153,7 @@ rules:
       resources:
           - egressfirewalls/status
           - adminpolicybasedexternalroutes/status
+          - egressqoses/status
```

Comment:
hmm why is the node receiving rbac to update eqos?

Reply:
this rbac is needed when node runs at its own ic zone and it would update qos status message.